README.md

```diff
@@ -1,17 +1,15 @@
 # Terragrunt
 
-Terragrunt is a thin wrapper for [Terraform](https://www.terraform.io/) that provides extra tools for working with
-multiple Terraform modules, remote state, and locking.
-
-
+Terragrunt is a thin wrapper for [Terraform](https://www.terraform.io/) that provides extra tools for keeping your Terraform scripts DRY, working with multiple Terraform modules, and managing remote state.
 
 
 ## Quick start
 
-1. Install [Terraform](https://www.terraform.io/).
+1. [Install Terraform](https://www.terraform.io/intro/getting-started/install.html).
 
-1. Install Terragrunt by going to the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases), downloading
-   the binary for your OS, renaming it to `terragrunt`, and adding it to your PATH.
+1. Install Terragrunt
+   * On OSX `brew install terragrunt`. 
+     See the [Install Terragrunt]() docs for other installation options.
```

Comment:
Ah ok. I'll make the changes at the same time that I lobby for keeping homebrew up to date ;) 

I don't know anything about creating homebrew packages. Is there a standard way to automate that to reduce maintenance overhead for each new release?